conda/recipes/meta.yaml

```diff
@@ -15,32 +15,41 @@
 # specific language governing permissions and limitations
 # under the License.
 
-{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
-{% set minor_version = version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set name = "datafusion" %}
+{% set major_minor_patch = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').split('.') %}
+{% set new_patch = major_minor_patch[2] | int + 1 %}
+{% set version = (major_minor_patch[:2] + [new_patch]) | join('.') + environ.get('VERSION_SUFFIX', '') %}
+
 
 package:
-  name: datafusion
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
   git_url: ../..
 
 build:
   noarch: python
```

Comment:
Do we have a preference for what platforms we want to build and publish for? Don't need to do those builds here, but if we intend to publish to conda-forge it's likely that we'll need to cross compile for Mac and Windows

Reply:
CI currently builds the following wheels that we upload to PyPi:

```
datafusion-22.0.0-cp37-abi3-macosx_10_7_x86_64.whl
datafusion-22.0.0-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
datafusion-22.0.0-cp37-abi3-win_amd64.whl
```

I am building the macos arm64 wheel manually but would love to get that automated.

Reply:
Yeah, I think that covers the main ones. Let me make another PR after this one for conda macOS arm64 builds. Will do that after this PR merges.